.github/workflows/node-ci.yml

```diff
@@ -97,6 +97,14 @@ jobs:
         files: ./coverage/coverage-final.json
         fail_ci_if_error: true
 
+    - uses: Khan/actions@check-for-changeset-v0
+      if: github.actor != 'dependabot[bot]' && github.actor != 'dependabot-preview[bot]'
+      with:
+        exclude: .github/,.storybook/
+
+    - name: Pre-publish check
+      run: node utils/pre-publish-check-ci.js
```

Comment:
Thank you. I was going to say add a comment about the size check going last, but I see there already is one. Could you add a similar comment after it saying NO MORE STEPS AFTER THIS STEP or something?

Reply:
I can do that. 🙂